### ISSUE TITLE
Memory is a tad more than expected on the memory module

### ISSUE THREAD

**Issue by sourjyo0** [tobi-wan-kenobi/bumblebee-status#594]
#### Summary
Affected module: memory

#### Description:
The memory and free -h ,htop commands are different
Usually the memory in status bar is more by about 200-300 MiB
Maybe 
`~$ free -h |awk '/^Mem:/{print $3"B/"$2"B"}'`
My screenshot:
![shadow](https://user-images.githubusercontent.com/58564906/78709826-f04a9700-7931-11ea-8e7b-c6492f847957.png)

might help
#### How to reproduce

Comparing the htop, free-h and memory module output

**Comment by sourjyo0** [comment]
`import os`
`os.system('free -h |awk \'/^Mem:/{print $3"B/"$2"B"}\'>temp ')`
`print(open("temp","r").read())\n`
Seems to work although Line 2 returns 0 :P

**Comment by somospocos** [comment]
Memory module reads the info from /proc/meminfo , if you suspect a bug in the memory module, you may want to match your data against that and have a look at bumblebee.modules.memory.Module.update() function to see how it gets calculated.

> Seems to work although Line 2 returns 0 :P

I suspect that's the return code of the command you're running with os.system()

**Comment by tobi-wan-kenobi** [comment]
Hi,

please have a look at #402, calculating the amount of available memory on a Linux system is non-trivial. Most tools (htop, gnome-system-monitor, etc.) report slightly different values, depending on their definition of "free".

In your particular case, using the column "free" in the output of "free -h" is misleading, if you have it, the column "available" would be better, from the man page:

Estimation of how much memory is available for starting new applications, without swap‐
ping. Unlike the data provided by the cache or free fields, this field takes into ac‐
count page cache and also that not all reclaimable memory slabs will be reclaimed due to
items being in use (MemAvailable in /proc/meminfo, available on kernels 3.14, emulated
on kernels 2.6.27+, otherwise the same as free)

In short, the column "free" does not for memory that is marked as used, but is reclaimable (and therefore, available to other applications).

If you still feel that the reported memory is incorrect, please reopen the ticket.

Thanks!
